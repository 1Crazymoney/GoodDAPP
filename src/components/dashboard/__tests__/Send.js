import React from 'react'
import { getWebRouterComponentWithMocks } from './__util__'
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'

describe('Send', () => {
  it('renders without errors', () => {
    const Send = getWebRouterComponentWithMocks('../Send')
    const tree = renderer.create(<Send />)
    expect(tree.toJSON()).toBeTruthy()
  })

  it('matches snapshot', () => {
    const Send = getWebRouterComponentWithMocks('../Send')
    const component = renderer.create(<Send />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
