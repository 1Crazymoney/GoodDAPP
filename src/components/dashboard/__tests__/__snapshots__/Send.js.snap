// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Send matches snapshot 1`] = `
<div
  className="css-view-1dbjc4n"
  style={
    Object {
      "WebkitBoxFlex": 1,
      "WebkitFlexBasis": "0%",
      "WebkitFlexGrow": 1,
      "WebkitFlexShrink": 1,
      "flexBasis": "0%",
      "flexGrow": 1,
      "flexShrink": 1,
      "msFlexNegative": 1,
      "msFlexPositive": 1,
      "msFlexPreferredSize": "0%",
    }
  }
>
  <div
    className="css-view-1dbjc4n"
    style={
      Object {
        "WebkitBoxFlex": 1,
        "WebkitFlexBasis": "0%",
        "WebkitFlexGrow": 1,
        "WebkitFlexShrink": 1,
        "display": "flex",
        "flexBasis": "0%",
        "flexGrow": 1,
        "flexShrink": 1,
        "msFlexNegative": 1,
        "msFlexPositive": 1,
        "msFlexPreferredSize": "0%",
      }
    }
  >
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "WebkitAlignItems": "stretch",
          "WebkitBoxAlign": "stretch",
          "WebkitBoxDirection": "normal",
          "WebkitBoxFlex": 1,
          "WebkitBoxOrient": "horizontal",
          "WebkitBoxPack": "center",
          "WebkitFlexBasis": "0%",
          "WebkitFlexDirection": "row",
          "WebkitFlexGrow": 1,
          "WebkitFlexShrink": 1,
          "WebkitJustifyContent": "center",
          "alignItems": "stretch",
          "display": "flex",
          "flexBasis": "0%",
          "flexDirection": "row",
          "flexGrow": 1,
          "flexShrink": 1,
          "justifyContent": "center",
          "msFlexAlign": "stretch",
          "msFlexDirection": "row",
          "msFlexNegative": 1,
          "msFlexPack": "center",
          "msFlexPositive": 1,
          "msFlexPreferredSize": "0%",
        }
      }
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "WebkitAlignItems": "stretch",
            "WebkitBoxAlign": "stretch",
            "WebkitBoxDirection": "normal",
            "WebkitBoxFlex": 1,
            "WebkitBoxOrient": "vertical",
            "WebkitBoxPack": "justify",
            "WebkitFlexBasis": "0%",
            "WebkitFlexDirection": "column",
            "WebkitFlexGrow": 1,
            "WebkitFlexShrink": 1,
            "WebkitJustifyContent": "space-between",
            "alignItems": "stretch",
            "display": "flex",
            "flexBasis": "0%",
            "flexDirection": "column",
            "flexGrow": 1,
            "flexShrink": 1,
            "justifyContent": "space-between",
            "msFlexAlign": "stretch",
            "msFlexDirection": "column",
            "msFlexNegative": 1,
            "msFlexPack": "justify",
            "msFlexPositive": 1,
            "msFlexPreferredSize": "0%",
            "paddingBottom": "1rem",
            "paddingLeft": "1rem",
            "paddingRight": "1rem",
            "paddingTop": "1rem",
            "width": "100%",
          }
        }
      >
        <div
          className="css-view-1dbjc4n"
          style={
            Object {
              "backgroundColor": "rgba(238,238,239,1.00)",
              "borderBottomLeftRadius": "5px",
              "borderBottomRightRadius": "5px",
              "borderTopLeftRadius": "5px",
              "borderTopRightRadius": "5px",
              "marginBottom": "15px",
              "paddingBottom": "15px",
              "paddingLeft": "10px",
              "paddingRight": "10px",
              "paddingTop": "15px",
            }
          }
        >
          <div
            className="css-view-1dbjc4n"
            style={
              Object {
                "WebkitAlignItems": "center",
                "WebkitBoxAlign": "center",
                "WebkitBoxDirection": "normal",
                "WebkitBoxOrient": "horizontal",
                "WebkitBoxPack": "justify",
                "WebkitFlexDirection": "row",
                "WebkitJustifyContent": "space-between",
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "space-between",
                "msFlexAlign": "center",
                "msFlexDirection": "row",
                "msFlexPack": "justify",
              }
            }
          >
            <div
              className="css-view-1dbjc4n"
            >
              <div
                className="css-view-1dbjc4n"
                style={
                  Object {
                    "backgroundColor": "rgba(255,255,255,1.00)",
                    "borderBottomLeftRadius": "17px",
                    "borderBottomRightRadius": "17px",
                    "borderTopLeftRadius": "17px",
                    "borderTopRightRadius": "17px",
                    "height": "34px",
                    "width": "34px",
                  }
                }
              >
                <div
                  className="css-view-1dbjc4n"
                  style={
                    Object {
                      "WebkitFlexBasis": "auto",
                      "borderBottomLeftRadius": "17px",
                      "borderBottomRightRadius": "17px",
                      "borderTopLeftRadius": "17px",
                      "borderTopRightRadius": "17px",
                      "flexBasis": "auto",
                      "height": "34px",
                      "msFlexPreferredSize": "auto",
                      "overflowX": "hidden",
                      "overflowY": "hidden",
                      "width": "34px",
                      "zIndex": 0,
                    }
                  }
                >
                  <div
                    className="css-view-1dbjc4n"
                    style={
                      Object {
                        "backgroundColor": "rgba(0,0,0,0.00)",
                        "backgroundPosition": "center",
                        "backgroundRepeat": "no-repeat",
                        "backgroundSize": "cover",
                        "bottom": "0px",
                        "height": "100%",
                        "left": "0px",
                        "position": "absolute",
                        "right": "0px",
                        "top": "0px",
                        "width": "100%",
                        "zIndex": -1,
                      }
                    }
                  />
                </div>
              </div>
            </div>
            <div
              className="css-view-1dbjc4n"
              style={
                Object {
                  "display": "inline-block",
                }
              }
            >
              <div
                className="css-text-76zvg2"
                dir="auto"
                style={
                  Object {
                    "color": "rgba(85,85,85,1.00)",
                    "direction": "ltr",
                    "fontFamily": "Helvetica, \\"sans-serif\\"",
                    "fontSize": "30px",
                    "textAlign": "right",
                  }
                }
              >
                0.00
              </div>
              <div
                className="css-text-76zvg2"
                dir="auto"
                style={
                  Object {
                    "color": "rgba(85,85,85,1.00)",
                    "direction": "ltr",
                    "fontFamily": "Helvetica, \\"sans-serif\\"",
                    "fontSize": "18px",
                    "textAlign": "right",
                  }
                }
              >
                GD
              </div>
            </div>
          </div>
        </div>
        <div
          className="css-view-1dbjc4n"
          style={
            Object {
              "WebkitBoxFlex": 1,
              "WebkitFlexBasis": "0%",
              "WebkitFlexGrow": 1,
              "WebkitFlexShrink": 1,
              "backgroundColor": "rgba(238,238,239,1.00)",
              "borderBottomLeftRadius": "5px",
              "borderBottomRightRadius": "5px",
              "borderTopLeftRadius": "5px",
              "borderTopRightRadius": "5px",
              "flexBasis": "0%",
              "flexGrow": 1,
              "flexShrink": 1,
              "marginBottom": "15px",
              "msFlexNegative": 1,
              "msFlexPositive": 1,
              "msFlexPreferredSize": "0%",
              "paddingBottom": "15px",
              "paddingLeft": "10px",
              "paddingRight": "10px",
              "paddingTop": "15px",
            }
          }
        >
          <div
            className="css-view-1dbjc4n"
            style={
              Object {
                "WebkitBoxFlex": 1,
                "WebkitFlexBasis": "0%",
                "WebkitFlexGrow": 1,
                "WebkitFlexShrink": 1,
                "flexBasis": "0%",
                "flexGrow": 1,
                "flexShrink": 1,
                "msFlexNegative": 1,
                "msFlexPositive": 1,
                "msFlexPreferredSize": "0%",
              }
            }
          >
            <div
              className="css-text-76zvg2"
              dir="auto"
              style={
                Object {
                  "color": "rgba(85,85,85,1.00)",
                  "fontFamily": "Helvetica, \\"sans-serif\\"",
                  "fontSize": "18px",
                  "textAlign": "center",
                }
              }
            >
              TO WHO?
            </div>
            <div
              className="css-view-1dbjc4n"
              style={
                Object {
                  "backgroundColor": "rgba(231,231,231,1.00)",
                  "borderTopLeftRadius": "4px",
                  "borderTopRightRadius": "4px",
                }
              }
            >
              <div
                className="css-view-1dbjc4n"
                style={
                  Object {
                    "WebkitTransform": "scaleY(0.5)",
                    "backgroundColor": "rgba(0,0,0,0.26)",
                    "bottom": "0px",
                    "height": "2px",
                    "left": "0px",
                    "position": "absolute",
                    "right": "0px",
                    "transform": "scaleY(0.5)",
                  }
                }
              />
              <input
                autoCapitalize="sentences"
                autoComplete="on"
                autoCorrect="on"
                className="css-textinput-1cwyjr8 r-placeholderTextColor-1qzd3f0"
                data-focusable={true}
                dir="auto"
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyPress={[Function]}
                onSelect={[Function]}
                readOnly={false}
                spellCheck={true}
                style={
                  Object {
                    "WebkitBoxFlex": 1,
                    "WebkitFlexGrow": 1,
                    "color": "rgba(0,0,0,1.00)",
                    "flexGrow": 1,
                    "fontFamily": "Roboto, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                    "fontSize": "16px",
                    "marginBottom": "0px",
                    "marginLeft": "0px",
                    "marginRight": "0px",
                    "marginTop": "0px",
                    "minHeight": "58px",
                    "msFlexPositive": 1,
                    "paddingBottom": "15px",
                    "paddingLeft": "12px",
                    "paddingRight": "12px",
                    "paddingTop": "15px",
                    "textAlign": "left",
                    "verticalAlign": "middle",
                    "zIndex": 1,
                  }
                }
                type="text"
              />
            </div>
            <div
              className="css-text-76zvg2"
              dir="auto"
              style={
                Object {
                  "WebkitTransform": "translateY(0px)",
                  "color": "rgba(176,0,32,1.00)",
                  "direction": "ltr",
                  "fontFamily": "Roboto, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                  "fontSize": "12px",
                  "opacity": 1,
                  "paddingBottom": "4px",
                  "paddingLeft": "12px",
                  "paddingRight": "12px",
                  "paddingTop": "4px",
                  "textAlign": "left",
                  "transform": "translateY(0px)",
                }
              }
            />
            <div
              className="css-view-1dbjc4n"
              style={
                Object {
                  "WebkitAlignItems": "center",
                  "WebkitBoxAlign": "center",
                  "WebkitBoxDirection": "normal",
                  "WebkitBoxOrient": "horizontal",
                  "WebkitBoxPack": "justify",
                  "WebkitFlexDirection": "row",
                  "WebkitJustifyContent": "space-between",
                  "alignItems": "center",
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                  "msFlexAlign": "center",
                  "msFlexDirection": "row",
                  "msFlexPack": "justify",
                }
              }
            >
              <div
                className="css-view-1dbjc4n"
                onClick={[Function]}
                style={
                  Object {
                    "WebkitAlignItems": "center",
                    "WebkitBoxAlign": "center",
                    "WebkitBoxDirection": "normal",
                    "WebkitBoxFlex": 1,
                    "WebkitBoxOrient": "horizontal",
                    "WebkitFlexBasis": "0%",
                    "WebkitFlexDirection": "row",
                    "WebkitFlexGrow": 1,
                    "WebkitFlexShrink": 1,
                    "alignItems": "center",
                    "cursor": "pointer",
                    "flexBasis": "0%",
                    "flexDirection": "row",
                    "flexGrow": 1,
                    "flexShrink": 1,
                    "msFlexAlign": "center",
                    "msFlexDirection": "row",
                    "msFlexNegative": 1,
                    "msFlexPositive": 1,
                    "msFlexPreferredSize": "0%",
                  }
                }
              >
                <div
                  className="css-view-1dbjc4n"
                  style={
                    Object {
                      "WebkitAlignItems": "center",
                      "WebkitBoxAlign": "center",
                      "WebkitBoxPack": "center",
                      "WebkitJustifyContent": "center",
                      "alignItems": "center",
                      "backgroundColor": "rgba(255,255,255,1.00)",
                      "borderBottomLeftRadius": "28px",
                      "borderBottomRightRadius": "28px",
                      "borderTopLeftRadius": "28px",
                      "borderTopRightRadius": "28px",
                      "height": "52px",
                      "justifyContent": "center",
                      "marginBottom": "7px",
                      "marginLeft": "7px",
                      "marginRight": "7px",
                      "marginTop": "7px",
                      "msFlexAlign": "center",
                      "msFlexPack": "center",
                      "width": "52px",
                    }
                  }
                >
                  <div
                    className="css-text-76zvg2"
                    dir="auto"
                    style={
                      Object {
                        "backgroundColor": "rgba(0,0,0,0.00)",
                        "color": "rgba(40,44,52,1.00)",
                        "fontFamily": "MaterialIcons",
                        "fontSize": "24px",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      }
                    }
                  >
                    
                  </div>
                </div>
                <div
                  className="css-text-76zvg2"
                  dir="auto"
                  style={
                    Object {
                      "color": "inherit",
                    }
                  }
                >
                  Scan QR Code
                </div>
              </div>
              <div
                className="css-view-1dbjc4n"
                onClick={[Function]}
                style={
                  Object {
                    "WebkitAlignItems": "center",
                    "WebkitBoxAlign": "center",
                    "WebkitBoxDirection": "normal",
                    "WebkitBoxFlex": 1,
                    "WebkitBoxOrient": "horizontal",
                    "WebkitFlexBasis": "0%",
                    "WebkitFlexDirection": "row",
                    "WebkitFlexGrow": 1,
                    "WebkitFlexShrink": 1,
                    "alignItems": "center",
                    "cursor": "pointer",
                    "flexBasis": "0%",
                    "flexDirection": "row",
                    "flexGrow": 1,
                    "flexShrink": 1,
                    "msFlexAlign": "center",
                    "msFlexDirection": "row",
                    "msFlexNegative": 1,
                    "msFlexPositive": 1,
                    "msFlexPreferredSize": "0%",
                  }
                }
              >
                <div
                  className="css-view-1dbjc4n"
                  style={
                    Object {
                      "WebkitAlignItems": "center",
                      "WebkitBoxAlign": "center",
                      "WebkitBoxPack": "center",
                      "WebkitJustifyContent": "center",
                      "alignItems": "center",
                      "backgroundColor": "rgba(255,255,255,1.00)",
                      "borderBottomLeftRadius": "28px",
                      "borderBottomRightRadius": "28px",
                      "borderTopLeftRadius": "28px",
                      "borderTopRightRadius": "28px",
                      "height": "52px",
                      "justifyContent": "center",
                      "marginBottom": "7px",
                      "marginLeft": "7px",
                      "marginRight": "7px",
                      "marginTop": "7px",
                      "msFlexAlign": "center",
                      "msFlexPack": "center",
                      "width": "52px",
                    }
                  }
                >
                  <div
                    className="css-text-76zvg2"
                    dir="auto"
                    style={
                      Object {
                        "backgroundColor": "rgba(0,0,0,0.00)",
                        "color": "rgba(40,44,52,1.00)",
                        "fontFamily": "MaterialIcons",
                        "fontSize": "24px",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      }
                    }
                  >
                    
                  </div>
                </div>
                <div
                  className="css-text-76zvg2"
                  dir="auto"
                  style={
                    Object {
                      "color": "inherit",
                    }
                  }
                >
                  Generate Link
                </div>
              </div>
            </div>
          </div>
          <div
            className="css-view-1dbjc4n"
          >
            <div
              className="css-view-1dbjc4n"
              style={
                Object {
                  "WebkitBoxPack": "center",
                  "WebkitJustifyContent": "center",
                  "backgroundColor": "rgba(0,0,0,0.12)",
                  "borderBottomColor": "rgba(0,0,0,0.00)",
                  "borderBottomLeftRadius": "4px",
                  "borderBottomRightRadius": "4px",
                  "borderBottomStyle": "solid",
                  "borderBottomWidth": "0px",
                  "borderLeftColor": "rgba(0,0,0,0.00)",
                  "borderLeftStyle": "solid",
                  "borderLeftWidth": "0px",
                  "borderRightColor": "rgba(0,0,0,0.00)",
                  "borderRightStyle": "solid",
                  "borderRightWidth": "0px",
                  "borderTopColor": "rgba(0,0,0,0.00)",
                  "borderTopLeftRadius": "4px",
                  "borderTopRightRadius": "4px",
                  "borderTopStyle": "solid",
                  "borderTopWidth": "0px",
                  "justifyContent": "center",
                  "minWidth": "64px",
                  "msFlexPack": "center",
                }
              }
            >
              <div
                aria-disabled={true}
                className="css-view-1dbjc4n"
                disabled={true}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                role="button"
                style={
                  Object {
                    "borderBottomLeftRadius": "4px",
                    "borderBottomRightRadius": "4px",
                    "borderTopLeftRadius": "4px",
                    "borderTopRightRadius": "4px",
                    "overflowX": "hidden",
                    "overflowY": "hidden",
                    "position": "relative",
                  }
                }
              >
                <div
                  className="css-view-1dbjc4n"
                  style={
                    Object {
                      "WebkitAlignItems": "center",
                      "WebkitBoxAlign": "center",
                      "WebkitBoxDirection": "normal",
                      "WebkitBoxOrient": "horizontal",
                      "WebkitBoxPack": "center",
                      "WebkitFlexDirection": "row",
                      "WebkitJustifyContent": "center",
                      "alignItems": "center",
                      "flexDirection": "row",
                      "justifyContent": "center",
                      "msFlexAlign": "center",
                      "msFlexDirection": "row",
                      "msFlexPack": "center",
                    }
                  }
                >
                  <div
                    className="css-text-76zvg2 css-textOneLine-bfa6kz"
                    dir="auto"
                    style={
                      Object {
                        "color": "rgba(0,0,0,0.32)",
                        "direction": "ltr",
                        "fontFamily": "Roboto-Medium, Roboto, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                        "letterSpacing": "1px",
                        "marginBottom": "9px",
                        "marginLeft": "16px",
                        "marginRight": "16px",
                        "marginTop": "9px",
                        "textAlign": "center",
                      }
                    }
                  >
                    <span
                      className="css-text-76zvg2 css-textHasAncestor-16my406"
                      dir="auto"
                      style={
                        Object {
                          "direction": "ltr",
                          "fontFamily": "Helvetica, \\"sans-serif\\"",
                          "fontSize": "18px",
                          "fontWeight": "bold",
                          "paddingBottom": "5px",
                          "paddingLeft": "5px",
                          "paddingRight": "5px",
                          "paddingTop": "5px",
                          "textAlign": "center",
                          "textTransform": "uppercase",
                        }
                      }
                    >
                      Continue
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
